Index: compat
===================================================================
--- compat	(revision 2428)
+++ compat	(working copy)
@@ -1,1 +1,1 @@
-7
+6
Index: control
===================================================================
--- control	(revision 2428)
+++ control	(working copy)
@@ -5,1 +6,1 @@
-Build-Depends: debhelper (>= 7), libgeos-dev, libxml2-dev, libpq-dev, dpatch, proj, libbz2-dev
+Build-Depends: debhelper (>= 6), libgeos-dev, libxml2-dev, libpq-dev, dpatch, proj, libbz2-dev
